File: GoChat/src/main/java/Lib/callback/GoChatCallBack.java
package Lib.callback;

/**
 * 创建时间：
 * 更新时间 2017/11/1 20:49
 * 版本：
 * 作者：Mr.jin
 * 描述：本项目通用回调
 */

public interface GoChatCallBack {

	void onSuccess();

	void onProgress(int progress);

	void onError(int error, String msg);

}

File: chat/src/main/java/com/jw/chat/callback/GoChatCallBack.kt
package com.jw.chat.callback

/**
 * 创建时间：
 * 更新时间 2017/11/1 20:49
 * 版本：
 * 作者：Mr.jin
 * 描述：本项目通用回调
 */

interface GoChatCallBack {
    fun onSuccess()
    fun onProgress(progress: Int)
    fun onError(error: Int, msg: String)
}